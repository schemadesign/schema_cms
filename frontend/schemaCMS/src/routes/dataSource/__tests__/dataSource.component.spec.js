import React from 'react';
import { shallow } from 'enzyme';

import DataSource from '../dataSource.component';

describe('DataSource: Component', () => {
  const defaultProps = {
    match: {
      path: 'path',
    },
  };

  const component = props => <DataSource {...defaultProps} {...props} />;

  const render = (props = {}) => shallow(component(props));

  it('should render correctly', () => {
    const wrapper = render();
    global.expect(wrapper).toMatchSnapshot();
  });
});
