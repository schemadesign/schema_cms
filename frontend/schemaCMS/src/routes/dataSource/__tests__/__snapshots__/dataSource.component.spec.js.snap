// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataSource: Component should render correctly 1`] = `
<Switch>
  <Route
    component={[Function]}
    exact={true}
    path="path/job"
  />
  <Redirect
    exact={true}
    path="path"
    to="path/1"
  />
  <Route
    component={[Function]}
    exact={true}
    path="path/:step"
  />
  <Route
    component={[Function]}
    exact={true}
    path="path/5/add"
  />
</Switch>
`;
